tools/github_helper/main.go

```diff
@@ -93,16 +93,50 @@ func newHelper(r *string) (*helper, error) {
 	}
 }
 
+// Create a new branch for fast forward
+func (h helper) createBranchForFastForward(commit *string) (*string, error) {
+	branchName := fmt.Sprintf("fastForward-%s-%s", h.Head, (*commit)[0:7])
+	ref := fmt.Sprintf("refs/heads/%s", branchName)
+
+	if refer, resp, err := h.Client.Git.GetRef(context.TODO(), h.Owner, h.Repo, ref); resp.StatusCode != 404 {
```

Comment:
I don t think you need to actually get the ref, you can just look at the reponse from CreateRef.